template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/resources/batch-inference-workflow-resource.yml.tmpl

```diff
@@ -3,6 +3,7 @@ new_cluster: &new_cluster
     num_workers: 3
     spark_version: 15.3.x-cpu-ml-scala2.12
     node_type_id: {{template `cloud_specific_node_type_id` .}}
+    data_security_mode: "SINGLE_USER"
```

Comment:
Can we confirm these workflows still run on AWS, Azure, and GCP?

Reply:
confirmed this works on dogfood(aws), will test the other in a bit